modules/theme/src/themes/wso2is/apps/login-portal.overrides

```diff
@@ -165,6 +165,20 @@
             }
         }
 
+        /** Helper class to support new line */
+        .new-line-support {
```

Comment:
How about `line-break` or `text-break` instead?

Reply:
Ack, will change it like that

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5423/commits/e445d5f7cf81e88dccfa4366bd555eba3a0648ba

Reply:
Also move this to `definitions/globals/product.less`

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5423/commits/ffa92cc66cc0e2a9a3b0d42f9768a7c31256a003